homeassistant/components/homekit/type_lights.py

```diff
@@ -54,9 +54,9 @@ def __init__(self, *args):
         super().__init__(*args, category=CATEGORY_LIGHTBULB)
 
         self.chars = []
-        self._features = self.hass.states.get(self.entity_id).attributes.get(
-            ATTR_SUPPORTED_FEATURES
-        )
+        state = self.hass.states.get(self.entity_id)
+
+        self._features = state.attributes.get(ATTR_SUPPORTED_FEATURES)
```

Comment:
Should you default to 0, in case the attribute doesn't exist?
```suggestion
        self._features = state.attributes.get(ATTR_SUPPORTED_FEATURES, 0)
```

Reply:
Yes